### ISSUE TITLE
Setup our own ontology RDF data to create examples, aka Basic Graph Patterns (BGPs)

### ISSUE THREAD

**Issue by firmao** [Mat-O-Lab/NaturalMSEQueries#2]
It supports the creation of Basic Graph Patterns([BGP](https://www.w3.org/2001/sw/DataAccess/rq23/sparql-defns.html#defn_BasicGraphPattern)) with the selection of values with autocomplete search or dropdown lists.
- One of the main first steps is to use our own data, e.g. [TeamMechanics data](https://raw.githubusercontent.com/Mat-O-Lab/RDFConverter/development/resources/franhoferTeamMechanik/data.ttl) and [BWMD ontology](http://rest.matportal.org/ontologies/BWMD-DOMAIN/submissions/1/download?apikey=66c82e77-ce0d-4385-8056-a95898e47ebb)
- In other words, adequate the left side of the screen to use our own ontology to build the BGPs.

To this aim the [documentation](https://docs.sparnatural.eu/index.html) would help with this task. Starting by "[Customize the configuration files JSON-LD or OWL configuration file](https://docs.sparnatural.eu/OWL-based-configuration)".
